tests/e2e/workloads/pgsql/test_pgsql_node_drain.py

```diff
@@ -42,15 +43,13 @@ def pgsql_setup(self, pgsql):
         self.sanity_helpers = Sanity()
 
     @pytest.mark.usefixtures(pgsql_setup.__name__)
-    def test_run_pgsql_node_drain(
-        self, pgsql, transactions=900, node_type='master'
-    ):
+    def test_run_pgsql_node_drain(self, pgsql):
         """
         Test pgsql workload
         """
         # Create pgbench benchmark
         pgsql.create_pgbench_benchmark(
-            replicas=3, transactions=transactions, clients=3
+            replicas=3, clients=3, transactions=1600
```

Comment:
please make sure that the change to the function signature of args ordering doesn't break existing usage